sequence_processing_pipeline/NuQCJob.py

```diff
@@ -202,6 +202,29 @@ def _move_helper(self, completed_files, regex, samples_in_project, dst):
         for fp in files_to_move:
             move(fp, dst)
 
+    @staticmethod
+    def _move_trimmed_files(project_name, output_path):
+        '''
+        Given output_path, move all fastqs to a new subdir named project_name.
+        :param project_name: The name of the new folder to be created.
+        :param output_path: The path to scan for fastq files.
+        :return: None
+        '''
+
+        # method made static and output_path made a parameter for easier
```

Comment:
should this be part of the docstring vs. a comment?

Reply:
Mostly there as a discussion point for us. Better to delete it after.